docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -0,0 +1,517 @@
+
+In step 1 of the generic procedure, configure the image set for mirroring as follows:
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:
+  registry:
+    imageURL: registry.to.mirror.to
+    skipTLS: false
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages:
+    - name: mta-operator
+      channels:
+      - name: stable-v7.0
+    - name: rhsso-operator
+      channels:
+      - name: stable
+  helm: {}
+----
+
+== Memory requirements for running {ProductShortName} on Red Hat OpenShift Local
+
+When installed on https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local], {ProductShortName} requires a minimum amount of memory to complete its analysis. Adding memory makes the analysis process run faster. The table below describes the {ProductShortName} performance with varying amounts of memory.
+
+.OpenShift Local {ProductShortName} memory requirements
+[cols="25%,75%", options="header"]
+|====
+|Memory (GiB)
+|Description
+
+
+|`10`
+|{ProductShortName} cannot run the analysis due to insufficient memory
+
+|`11`
+|{ProductShortName} cannot run the analysis due to insufficient memory
+
+|`12`
+|*{ProductShortName} works and the analysis is completed in approximately 3 minutes*
+
+|`15`
+|{ProductShortName} works and the analysis is completed in less than 2 minutes
+
+|`20`
+|{ProductShortName} works quickly, and the analysis is completed in less than 1 minute
+|====
+
+The test results indicate that the minimum amount of memory for running {ProductShortName} on OpenShift Local is *12 GiB*.
+
+[NOTE]
+====
+** The tests were performed by running the {ProductShortName} binary analysis through the {WebName}.
+** All the analyses used the `tackle-testapp` binary.
+** All the tests were conducted on an OpenShift Local cluster without the monitoring tools installed.
+** Installing the cluster monitoring tools requires an additional 5 GiB of memory.
+====
+
+[id="eviction-threshold_{context}"]
+=== Eviction threshold
+
+Each node has a certain amount of memory allocated to it. Some of that memory is reserved for system services. The rest of the memory is intended for running pods. If the pods use more than their allocated amount of memory, an out-of-memory event is triggered and the node is terminated with a `OOMKilled` error.
+
+To prevent out-of-memory events and protect nodes, use the `--eviction-hard` setting. This setting specifies the threshold of memory availability below which the node evicts pods. The value of the setting can be absolute or a percentage.
+
+.Example of node memory allocation settings
+
+- Node capacity: `32 GiB`
+
+- `--system-reserved` setting: `3 GiB`
+
+- `--eviction-hard` setting: `100 MiB`
+
```

Comment:
@sbeskin-redhat - please can I ask if you can see any issue with this block

I am getting a warning:

```
asciidoctor: WARNING: topics/mta-7-installing-web-console-on-openshift.adoc: line 260: invalid style for listing block: sample
```
I think I am ok to skip, but would appreciate a second opinion